hedera-node/configuration/testnet/bootstrap.properties

```diff
@@ -1,5 +1,10 @@
-ledger.id=0x01
-contracts.chainId=296
-bootstrap.genesisPublicKey=e06b22e0966108fa5d63fc6ae53f9824319b891cd4d6050dbf2b242be7e13344
```

Comment:
Are these not also different in testnet from mainnet?

Reply:
Yes, I reverted them back to what they are. Not sure why I can't see them on file 121.